### PR TITLE
Add bearer token option (--token) to cli

### PR DESCRIPTION
**Pull request by pieper**
This allows access to Google DICOMweb stores
as shown in the new cli example.

### CODE REVIEW

File: src/dicomweb_client/cli.py [hunk 1]
```diff
@@ -443,6 +447,13 @@ def _print_pixel_data(pixels):
     print(pixels)
     print('\n')
 
+def _headers(args):
+    headers = None
+    if hasattr(args, "token"):
+        headers = {
+            "Authorization" : "Bearer %s" % args.token
```
Comment (hackermd) [inline src/dicomweb_client/cli.py]: Use Python 3 style format string instead

**Comment by hackermd** [comment]
I recently added support for streaming data using chunked transfer encoding (see branch "feature/chunked-transfer-encoding"). As part of this effort, I also refactored the command line interface, such that the client is constructed in the ``main()`` function and then passed to other functions:
https://github.com/MGHComputationalPathology/dicomweb-client/blob/462f6a587af88bc220313f693edcb2570fc5373c/src/dicomweb_client/cli.py#L638-L645
This avoids having to construct the client in each function separately with the same set of passed arguments.

You may want to consider re-basing your changes onto this feature branch instead of the current master branch.

**Comment by pieper** [comment]
Probably it's easiest to wait until you merge your chunked transfer branch into master and then I'll rebase.  I do like the idea of unifying the client creation code.

**Comment by hackermd** [comment]
> Probably it's easiest to wait until you merge your chunked transfer branch into master and then I'll rebase. I do like the idea of unifying the client creation code.

I have already tested the chunked transfer branch with a dcm4chee-arc-light origin server but would like to add a couple of unit tests before merging it into master.

In addition, I would like to refactor the code a bit under the hood to allow streaming data sets directly to and from files on disk. However, this could wait and get addressed with a subsequent release.

**Comment by hackermd** [comment]
@pieper would you mind testing the chunked transfer branch against the Google Healthcare API?

**Comment by pieper** [comment]
It was easier just to create a new PR.

**Pull request closed without merging**